molior-deploy

```diff
@@ -1110,6 +1113,15 @@ docker_install_cleanup()
   exit_on_error "Error mounting $target/sys"
 }
 
+check_required_version()
+{
+    if [ "$REQUIRED_VERSION" != "" ]; then
+        log "Checking required version ..."
+        dpkg --compare-versions $REQUIRED_VERSION eq $MOLIOR_TOOLS_VERSION
```

Comment:
Do you really want to check for equality or greater than?

Reply:
Or maybe even better something like `REQUIRED_VERSION=">= x.x.x"` or `REQUIRED_VERSION="x.x.x"`

Reply:
To be backward compatible we really need to stick to specific version. That's the reason why I would prefer to have equal. We can maybe also add MINIMUM_VERSION if necessary (I talked about it with @neolynx ). For now, REQUIRED_VERSION shall be equal to MOLIOR DEPLOY version.

Reply:
The thing I am saying is that it would be nice to parametrize the `op` parameter as well:

```
$ man dpkg
...
       --compare-versions ver1 op ver2
              Compare version numbers, where op is a binary operator. dpkg returns true (0) if the specified condition is satisfied, and false (1) otherwise. There are two groups of operators, which differ in how they treat an empty  ver1  or  ver2.  These  treat  an  empty
              version  as  earlier than any version: lt le eq ne ge gt. These treat an empty version as later than any version: lt-nl le-nl ge-nl gt-nl. These are provided only for compatibility with control file syntax: < << <= = >= >> >. The < and > operators are obsolete
              and should not be used, due to confusing semantics. To illustrate: 0.1 < 0.1 evaluates to true.
...
```

So one could be a bit less restricted.

Reply:
I think it is a bit strict to force a version, but I would be happy with an additional MINIMUM_MOLIORTOOLS_VERSION :)

I would expect user to use the latest version always, and molior-tools should be backward compatible anyway...

Reply:
REQUIRED_MOLIORTOOLS_VERSION="lt X.Y.Z" works. I will change to support such format so one can then specify operator (=, lt, le, gt, ge,...) directly in variable.